src/assisted_test_infra/test_infra/controllers/tang_controller/tang_controller.py

```diff
@@ -0,0 +1,55 @@
+import os
+import shutil
+import socket
+
+import consts
+from assisted_test_infra.test_infra import utils
+from service_client import log
+
+
+class TangController:
```

Comment:
either on module or class scope, we'll need some docstring to explain this class purpose